a10_octavia/db/repositories.py

```diff
@@ -350,6 +350,13 @@ def get_lb_count_by_flavor(self, session, project_id, flavor_id):
                 or_(self.model_class.provisioning_status == consts.PENDING_DELETE,
                     self.model_class.provisioning_status == consts.ACTIVE)).count()
 
+    def get_hmt_lb_count_by_flavor(self, session, project_ids, flavor_id):
```

Comment:
Can we not update the function "get_lb_count_by_flavor" to fetch flavors according to a project list. Now we have 2 functions doing a similar thing.

Reply:
Do you mean not create another API but just modify current get_lb_count_by_flavor() to support project_ids?

Reply:
yes, I think the purpose of both functions is to get the lb count for flavor.

Reply:
done